Readme.md

```diff
@@ -0,0 +1,47 @@
+# wedding project
+
+this project is all about online invitation.
```

Comment:
Give a description that can make someone want to visit the site or have it on their local machine